src/store/eventing/middleware.ts

```diff
@@ -22,7 +20,6 @@ const eventHandlers: EventHandlerMap = {
     [fetchAnnotationsAction.rejected.toString()]: handleFetchErrorEvents,
     [setActiveAnnotationIdAction.toString()]: handleActiveAnnotationEvents,
     [setIsInitialized.toString()]: handleAnnotationsInitialized,
-    [toggleAnnotationModeAction.toString()]: handleToggleAnnotationModeAction,
```

Comment:
Are these changes related to this PR? I think we should try to follow the "one change per PR" rule.